components/ui/ace-editor.vue

```diff
@@ -76,6 +76,7 @@ export default {
       })
     },
     lang(value) {
+      console.log("ace/mode/" + value)
```

Comment:
Can we remove this - or add a prefix such as `Current editor:`?
```suggestion
      console.log("[INFO] Current Editor: ace/mode/" + value)
```

```suggestion
```

Reply:
Umm, that was a log I forgot to remove while updating Ace Editor component code, I am committing the removal of that log.